examples/templates/render/hello_jupiter/README.md

```diff
@@ -6,21 +6,27 @@ by doing a simple string replacement.
 
 To run this, cd to the root of this git repo, then run these steps:
 
-1. First, compile the `abc` binary:
+1. Compile the `abc` binary:
 
     ```shell
     $ go build
     ```
 
-2. Now execute the template defined by the `spec.yaml` file in the example directory.
+1. (Optional) Install shell completions:
```

Comment:
I'd rather leave this out of the examples, and put it in the (eventual) user quickstart docs. I think it's important for this example to be as utterly simple as possible. I added a note to #29 so we don't forget.

Reply:
Done